src/js/components/Text/text.stories.js

```diff
@@ -30,6 +32,22 @@ const Color = () => (
   </Grommet>
 );
 
+const WordBreak = () => (
```

Comment:
Thank you for adding a storybook example

Reply:
Welcome